status.go

```diff
@@ -432,7 +432,6 @@ func (vs *version) status() versionStatus {
 		CreatedAt:  vs.created.UTC().Truncate(time.Second),
 		State:      vs.state,
 		Partitions: partitions,
-		ShardID:    vs.sequins.config.Sharding.ShardID,
```

Comment:
Probably should have put this in another PR, but I forgot to flag this in yours 😅 

This misses a lot of cases where the ShardID can be set (self-assigned id file, smallest missing, etc) and the Shard ID is already exposed in the header of requests so I feel like this isn't needed. Unless you explicitly wanted to expose what was set in the config file?